changelogs/fragments/7694-Keycloak-components-info.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - keycloak utils - retrive JSON representation of components (https://github.com/ansible-collections/community.general/pull/7694).
```

Comment:
I think this can be removed, since the module utils are not modified in this PR.